spec/lib/hypgen/worker/runner_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe Hypgen::Worker::Runner do
+  it 'start all experiment parts' do
+    workflow_class = double
+    workflow = instance_double("Hypgen::Workflow", set_set_id: true)
+
+    planner_class = double
+    planner = instance_double("Hypgen::Planner")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.